### ISSUE TITLE
root node lost when using renderToJson with find

### ISSUE THREAD

**Issue by yesmeck** [adriantoine/enzyme-to-json#30]
This issue may related to #17.

Following test:

```javascript
const App = () => (
  <div>
    <ul>
      <li>0</li>
      <li>1</li>
    </ul>
  </div>
);

const wrapper = render(App);
expect(renderToJson(wrapper.find('ul'))).toMatchSnapshot();
```

Generated snapshot:

```
<li>0</li>
```

I'd expect the generated snapshot contains the whole `ul` node.

**Comment by MateuszJonak** [comment]
This is because of this line: 
https://github.com/adriantoine/enzyme-to-json/blob/master/src/render.js#L18

**Comment by yesmeck** [comment]
I know this line cause the problem. But why?

**Comment by MateuszJonak** [comment]
I don't know but I have a problem with this too.

**Comment by wuct** [comment]
I've also run into similar issue with `mount`.

For example: 

```JS
const App = () => (
  <div>
    <ul>
      <li>0</li>
      <li>1</li>
    </ul>
  </div>
);

const wrapper = mount(App);
```

When I call

```JS
expect(renderToJson(wrapper.find('li'))).toMatchSnapshot();
```

I expects to get something like

```JS
Array[
  <li>0</li>,
  <li>1</li>
]
```

but I get

```JS
<li>0</li>
```

It's because [this line](https://github.com/adriantoine/enzyme-to-json/blob/master/src/mount.js#L54)

We can change this behavior by checking the length of the wrapper first. In my mind, it would be something like:

```js
if (wrapper.length > 1) {
  wrapper.nodes.map(toJson)
} else {
  toJson(wrapper.node)
}
```

I would like to create a PR with related tests if this looks good to you.

**Comment by adriantoine** [comment]
Hi, sorry for the delay, I will have a look!

**Comment by adriantoine** [comment]
This is fixed by #45 and will be released in the next patch version!
